.github/workflows/linting-and-tests.yml

```diff
@@ -287,6 +287,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Collect Workflow Telemetry
+        uses: runforesight/workflow-telemetry-action@v1
```

Comment:
this is useful to see how much CPU/memory various steps in the workflow are using. It attaches an artifact to the build that looks something like this:
<img width="791" alt="Screenshot 2023-05-23 at 19 33 13" src="https://github.com/grafana/oncall/assets/9406895/3c0f38d6-c719-4ba6-a300-838c20a9d218">
<img width="695" alt="Screenshot 2023-05-23 at 19 33 20" src="https://github.com/grafana/oncall/assets/9406895/31fedf6e-3a20-4257-8a09-ba22216de9be">